lib/callbacks.js

```diff
@@ -90,6 +90,21 @@ const unsafeFunction = fn => (typeof fn === 'function' ? fn : null);
 // Returns: <Function>, function or `common.emptiness` if fn is not a function
 const safeFunction = fn => (typeof fn === 'function' ? fn : emptiness);
 
+// Identity function
+//   x - <any>, incoming value which will be returned
+// Returns: <any>, incoming value
+const id = x => x;
+
+// Async identity function
+//   x - <any>, incoming value which will be returned into the callback
+//   callback - <Function>, callback to be called with first argument
+//     err - <null>
+//     data - <any>
+const asyncId = (x, callback) => {
```

Comment:
I'm not sure about the usage of this function. I thought this would be some sort of callback-identity to be used when there is no need in a specific callback but the implementation requests one or there are multiple arguments returned but you only need the first one.
i.e.
```js
const asyncId = callback => (err, x) => {
  if (err) callback(err);
  else callback(null, x);
};
```